packages/api/src/coinbase.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  getUSDRate: async () => {
+    const response = await fetch('https://api.coinbase.com/v2/exchange-rates?currency=DASH', {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json'
+      }
+    })
+
+    if (response.status !== 200) {
+      throw new Error('Coinbase api error')
```

Comment:
I'm afraid it could be hard to debug. Can we put some more additional information like response.text() and response.statusCode?